packages/dapp/src/utils/eth/contracts.js

```diff
@@ -0,0 +1,17 @@
+import { Contract } from 'ethers'
```

Comment:
I removed the functions that are now being handled via wagmi inside of `Footer.js`

Reply:
That seems like a bad idea given the hack around getting the chainId now...

Reply:
Still thinking about it 🤔